hrms/api/employee.py

```diff
@@ -0,0 +1,33 @@
+import fastapi as fa
+from hrms.svc import service
+import typing as ty
+from pydantic import BaseModel
```

Comment:
Let's make a habit of forming import statements in the following manner.
1. Only module should be imported and not other symbols like class, object, etc. i.e. `import pydantic as pyd`
2. If you have to import from sub-packages, try `from package.subpackage import module as mod`